### ISSUE TITLE
Accessibility pass

### ISSUE THREAD

**Issue by Tourmi** [Tourmi/Dev-Blog#7]
Grab a screen reader, and make sure the site is usable.
Make sure the palette colors are fine for colorblind people
Other accessibility stuff should be looked up
